src/DependencyInjection/CmfRoutingExtension.php

```diff
@@ -62,7 +62,7 @@ private function setupFormTypes(array $config, ContainerBuilder $container, Load
     {
         $loader->load('form-type.xml');
 
-        if (isset($config['dynamic'])) {
+        if (array_key_exists('dynamic', $config)) {
```

Comment:
the problem with isset is that the following code would not lead to an error:

```
function foo(array $config) 
{
    if (isset($konfig['dynamic'])) {
        ....
```

that is, typos in variable names get ignored. i found this to be quite a hassle in refactorings.